js_modules/dagster-ui/packages/ui-core/src/app/CloudOSSContext.tsx

```diff
@@ -14,7 +14,7 @@ type FeatureContext = {
 export const CloudOSSContext = React.createContext<{
   isBranchDeployment: boolean;
   featureContext: FeatureContext;
-  useAugmentSearchResults: () => (results: SearchResult[]) => SearchResult[];
+  useAugmentSearchResults: () => (results: SearchResult[], isCatalog: boolean) => SearchResult[];
```

Comment:
I'm always confused by boolean args. Maybe a union of possibilities, even if there's only two of them? e.g. `searchContext: 'catalog' | 'global-search'`?